com.microsoft.mrtk.core/Configuration/Default Profiles/MRTKProfile.asset

```diff
@@ -14,6 +14,8 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   isCustomProfile: 0
   loadedSubsystems:
+  - reference: Microsoft.MixedReality.Toolkit.Speech.Windows.WindowsTextToSpeechSubsystem,
```

Comment:
Is there a reason why we're adding this as a loaded subsystem in our default profile?